src/shared/scrapers/CA/index.js

```diff
@@ -94,7 +125,8 @@ const scraper = {
         throw new Error(`Timeseries does not contain a sample for SCRAPE_DATE ${scrapeDateString}`);
       }
 
-      regions.push(transform.sumData(regions));
+      // no need to do this as they provide as the summary data
```

Comment:
I've been asserting the summed === the total scraped from the site, may be useful, not sure.

Reply:
Oh, that's a good idea to check with assert!